domains/zeto/pkg/types/abi.go

```diff
@@ -61,13 +61,15 @@ type NonFungibleMintParams struct {
 type FungibleMintParams struct {
 	Mints []*FungibleTransferParamEntry `json:"mints"`
 }
+
 type FungibleTransferParams struct {
 	Transfers []*FungibleTransferParamEntry `json:"transfers"`
 }
 
 type FungibleTransferParamEntry struct {
 	To     string               `json:"to"`
 	Amount *pldtypes.HexUint256 `json:"amount"`
+	Data   pldtypes.HexBytes    `json:"data"`
```

Comment:
If I understand correct, this change will be a breaking API change as ABI does not have the concept of optional parameters.

Reply:
Correct